src/views/product-downloads-view/components/page-header/index.tsx

```diff
@@ -52,10 +47,13 @@ const PageHeader = ({
 				>
 					{pageTitle}
 				</Heading>
-				<Text className={s.pageHeaderSubtitle} size={300} weight="regular">
-					{pageSubtitle}
-				</Text>
 			</div>
+			<VersionContextSwitcher
+				onChange={(e: SyntheticEvent) =>
+					setCurrentVersion(String(e.currentTarget))
```

Comment:
<img width="1183" alt="Screenshot 2023-11-03 at 4 22 12 PM" src="https://github.com/hashicorp/dev-portal/assets/36613477/7b43de3c-a726-4f7b-964e-f57cdaf155c2">

Seeing an error on the preview when selecting a new version.

Reply:
Fixed!